proto-gen/api_v2/service.pb.go

```diff
@@ -0,0 +1,2062 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
`service.pb.go` is an odd name

Reply:
yeah, I'd originally tried `query.proto` but `protoc` complains about filenames shadowing one another with the existing `query.proto`, which is why I chose `service.proto` but agree it could be better named.

I've gone with `metricsquery.proto`. Let me know if you prefer another name.